dev-tools/kubernetes/Tiltfile

```diff
@@ -103,4 +103,3 @@ beat(beat="metricbeat", # change to `filebeat` to test filebeat
 
 
 
-
```

Comment:
@ChrsMark I am not sure about this change. I have copy/pasted the file from `main` branch. it has 106 lines but for some reason this change says that I deleted an empty line here

Reply:
hmm, I don't get it. Do you want to delete the line or not in this PR? The change says that you are deleting one line. If it's not intentional then it might be your IDE that is removing it. In any case, it's not a big deal :), you can just remove it here.

Reply:
I don't get it. the file in Main has 105. The file on my laptop has 105 lines. And it complains that I am deleting an extra line to have 104 lines.